config/detekt.yml

```diff
@@ -0,0 +1,751 @@
+build:
```

Comment:
My recommendation is to not have a full config file but an addition/override config on top of the standard one so that it can be easily maintained

Reply:
I like the idea, but doesn't seem there's an easier way of doing it than https://detekt.dev/docs/introduction/extensions/, which we already did in `airwallex-detekt-rules`